test/util/cluster/cluster.go

```diff
@@ -115,8 +116,13 @@ func (n *EnabledNodes) FindOneMellanoxSriovDevice(node string) (*sriovv1.Interfa
 func SriovStable(operatorNamespace string, clients *testclient.ClientSet) (bool, error) {
 	nodeStates, err := clients.SriovNetworkNodeStates(operatorNamespace).List(context.Background(), metav1.ListOptions{})
 	if err != nil {
+		switch err {
```

Comment:
This can be a single switch, without the extra nesting.
```
switch {
	err == o.ErrUnexpectedEOF:
		
	case err != nil:

	}
```

Reply:
Done